targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/TARGET_MCU_NRF52840/CMakeLists.txt

```diff
@@ -25,6 +25,13 @@ target_include_directories(mbed-ep-atlas
         TARGET_EP_ATLAS
 )
 
+target_sources(mbed-ep-atlas
```

Comment:
you need to create the library `add_library(mbed-ep-atlas INTERFACE)` and possibly later here to link it with nrf52xx library to get common drivers,sdks, etc.

Reply:
Ah wait, it's not visible in my view, but I can see it already there